close_test.go

```diff
@@ -415,15 +415,15 @@ func TestCloseOneSide(t *testing.T) {
 // a connection is closed, and ensures there's no race conditions such as the error
 // frame being added to the channel just as it is closed.
 func TestCloseSendError(t *testing.T) {
-	closed := uint32(0)
-	counter := uint32(0)
+	closed := &atomic.Uint32{}
```

Comment:
i'd like both of these to just be
```go
var (
  counterr atomic.Uint32
  closed atomic.Uint32
)
```

or if we want to be explicit about the 0 value, keep it similar to what we have now
```
closed := atomic.NewUint32(0)
counter := atomic.NewUint32(0)
```

since we're not passing them anywhere, we should't need a pointer